src/Raven.Server/Documents/ETL/Providers/Elasticsearch/ElasticsearchEtl.cs

```diff
@@ -0,0 +1,160 @@
+        {
+            throw new NotSupportedException("Counters aren't supported by ELASTICSEARCH ETL");
+        }
+
+        protected override IEnumerator<ElasticsearchItem> ConvertTimeSeriesEnumerator(DocumentsOperationContext context, IEnumerator<TimeSeriesSegmentEntry> timeSeries,
+            string collection)
+        {
+            throw new NotSupportedException("Time series aren't supported by ELASTICSEARCH ETL");
+        }
+
+        protected override IEnumerator<ElasticsearchItem> ConvertTimeSeriesDeletedRangeEnumerator(DocumentsOperationContext context,
+            IEnumerator<TimeSeriesDeletedRangeItem> timeSeries, string collection)
+        {
+            throw new NotSupportedException("Time series aren't supported by ELASTICSEARCH ETL");
+        }
+
+        protected override EtlTransformer<ElasticsearchItem, ElasticsearchIndexWithRecords, EtlStatsScope, EtlPerformanceOperation> GetTransformer(DocumentsOperationContext context)
+        {
+            return new ElasticsearchDocumentTransformer(Transformation, Database, context, Configuration);
+        }
+
+        protected override int LoadInternal(IEnumerable<ElasticsearchIndexWithRecords> records, DocumentsOperationContext context, EtlStatsScope scope)
+        {
+            Uri[] nodes = Configuration.Connection.Nodes.Select(x => new Uri(x)).ToArray();
+            var pool = new StaticConnectionPool(nodes);
+            var settings = new ConnectionSettings(pool);
+            var client = new ElasticClient(settings);
+            int statsCounter = 0;
+            
+            foreach (var index in records)
+            {
+                StringBuilder deleteQuery = new StringBuilder();
+                
+                foreach (ElasticsearchItem delete in index.Deletes)
+                {
+                    deleteQuery.Append($"{delete.DocumentId},");
+                }
+
+                var deleteResponse = client.DeleteByQuery<string>(d => d
+                    .Index(index.IndexName.ToLower())
+                    .Query(q => q
+                        .Match(p => p
+                            .Field(index.IndexIdProperty)
+                            .Query($"{deleteQuery}"))
+                    )
+                );
+
+                if (deleteResponse.ServerError != null)
+                {
+                    throw new Exception(deleteResponse.ServerError.Error.ToString());
+                }
+
+                statsCounter += (int)deleteResponse.Deleted;
+
+                foreach (ElasticsearchItem insert in index.Inserts)
+                {
+                    if (insert.Property == null) continue;
+
+                    var response = client.LowLevel.Index<StringResponse>(
+                        index: index.IndexName.ToLower(),
+                        body: insert.Property.RawValue.ToString(), requestParameters: new IndexRequestParameters(){Refresh = Refresh.WaitFor});
+
+                    if (!response.Success)
```

Comment:
Minor: please convert to `if (response.Success == false)`. We always use this notation